crowbar_framework/app/assets/stylesheets/content/animation.scss

```diff
@@ -0,0 +1,10 @@
+.led.unready,
+.led.spin,
+.led.in_process {
+  animation: play 0.8s steps(8) infinite;

```

Comment:
This is still not using the bootstrap mixin for animations, e.g. `@include animation(play .8s steps(8) infinite)`